docs/.vuepress/config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  title: "REI CO-OP Design System",
+  description: "A demo documentation using VuePress",
+  head: [
+    ['link', { rel: 'stylesheet', href:'https://fonts.googleapis.com/css?family=Roboto+Condensed:400|Roboto:400' }]
+  ],
+  themeConfig: {
+    logo: '/doc-site-logo.png',
+    search: false,
+    nav: [
```

Comment:
I think it's best that we use the built-in Vuepress conventions as much as possible. In this case, I suggest using **sidebar** property for this navigation instead of **nav**. **nav** by default is meant to be used for global top level nav, but our global nav as of now is through a sidebar. The documentation for specifying **sidebar** can be found here [https://vuepress.vuejs.org/default-theme-config/#sidebar](url)